AutoRest/Generators/CSharp/CSharp/Templates/MethodTemplate.cshtml

```diff
@@ -52,9 +62,9 @@ public async Task<@(Model.OperationResponseReturnTypeString)> @(Model.Name)WithH
     {
         invocationId = ServiceClientTracing.NextInvocationId.ToString();
         Dictionary<string, object> tracingParameters = new Dictionary<string, object>();
-    @foreach (var parameter in Model.LocalParameters)
+    @foreach (var parameter in Model.ParameterTemplateModels.Where(p => !p.IsParameterGroup && p.ClientProperty == null))
```

Comment:
you have already updated the "LocalParameters" getter, so filter out one with non empty "ParameterGroup", to me the check of"IsParameterGroup" here is redundant.